src/locales/ko/mystery-encounters/teleporting-hijinks-dialogue.json

```
@@ -1,27 +1,27 @@
 {
-  "intro": "It's a strange machine, whirring noisily...",
-  "title": "Teleportating Hijinks",
-  "description": "The machine has a sign on it that reads:\n  \"To use, insert money then step into the capsule.\"\n\nPerhaps it can transport you somewhere...",
-  "query": "What will you do?",
+  "intro": "시끄럽게 윙윙 소리가 나는 이상한 기계를 발견했다….",
+  "title": "텔레포트 대소동",
+  "description": "기계에는 다음과 같은 표지판이 있습니다:\n  \"사용하려면, 돈을 넣고 캡슐에 들어가세요.\"\n\n어딘가로 당신을 데려다줄 수 있을겁니다…….",
+  "query": "어떻게 하시겠습니까?",
   "option": {
     "1": {
-      "label": "Put Money In",
-      "tooltip": "(-) Pay {{price, money}}\n(?) Teleport to New Biome",
-      "selected": "You insert some money, and the capsule opens.\nYou step inside..."
+      "label": "돈을 넣는다",
+      "tooltip": "(-) {{price, money}} 지불\n(?) 다른 바이옴으로 이동",
+      "selected": "돈을 넣자, 캡슐이 열렸다.\n당신은 안으로 들어갔다……."
     },
     "2": {
-      "label": "A Pokémon Helps",
-      "tooltip": "(-) {{option2PrimaryName}} Helps\n(+) {{option2PrimaryName}} gains EXP\n(?) Teleport to New Biome",
-      "disabled_tooltip": "You need a Steel or Electric Type Pokémon to choose this",
-      "selected": "{{option2PrimaryName}}'s Type allows it to bypass the machine's paywall!$The capsule opens, and you step inside..."
+      "label": "포켓몬의 도움받기",
+      "tooltip": "(-) {{option2PrimaryName}}[[가]] 도와줌\n(+) {{option2PrimaryName}}[[가]] 경험치를 얻음\n(?) 다른 바이옴으로 이동",
+      "disabled_tooltip": "강철 또는 전기 타입 포켓몬이 있어야한다.",
+      "selected": "{{option2PrimaryName}}의 타입으로 기계의 지불 시스템을 해제했다!$캡슐이 열리자, 당신은 안으로 들어갔다…."
```

```suggestion
      "selected": "{{option2PrimaryName}}의 타입 덕분에 기계의 지불 시스템을 해제할 수 있었다!$캡슐이 열리자 당신은 안으로 들어갔다…."
```
'타입' 자체를 직접 써서 해제했다고 하기보다는 그 타입이 지닌 여러 능력/특성으로써 해제했다는 뉘앙스 같아서 약간 고쳐 보았습니다. 이 부분은 좀 더 논의해 보면 좋을 듯합니다.

---

전기랑 강철타입만되는데 그냥 부쉈다는 느낌이라서 그렇게 된거같아요ㅋㅋ
이건 반영해둘게요